app/controllers/users_controller.rb

```diff
@@ -1,15 +1,23 @@
 class UsersController < ApplicationController
   def accept_transition_info
-    accept_screen("accept_transition_screen", Settings.rollover ? rollover_path : providers_path)
+    update_params = { state: next_state }
+    update_state(Settings.rollover ? rollover_path : providers_path, update_params)
   end
 
   def accept_rollover
-    accept_screen("accept_rollover_screen", providers_path)
+    update_params = { state: next_state }
+    update_state(providers_path, update_params)
+  end
+
+  def accept_accredited_body_new_features
```

Comment:
These controller actions have very similar logic. Do we want to consolidate to have one action or keep separate (i.e. is it important for us to track the different actions via different end points?)